projects/elife.yaml

```diff
@@ -773,7 +773,7 @@ generic-cdn:
             surrogate-keys:
                 article-id:
                     url: "^/articles/([0-9]+)/(.+)$"
-                    value: "articles/\\1"
+                    value: "articles/\\1 article/\\1"
```

Comment:
Will define `version/...` here too, but there are file like `/articles/10627/elife-10627-v1-download.pdf` that makes it a bit more convoluted. Are those still used?